Examples/test-suite/ocaml/Makefile.in

```diff
@@ -85,9 +86,24 @@ include $(srcdir)/../common.mk
 	$(setup)
 	$(run_testcase)
 
+swig.cmi:
+	env SWIG_LIB=$(SWIG_LIB_DIR) $(SWIGEXE) -ocaml -co swig.mli
+	$(OCC) -c swig.mli
+swig.cmo:
+	env SWIG_LIB=$(SWIG_LIB_DIR) $(SWIGEXE) -ocaml -co swig.ml
+	$(OCC) -c swig.ml
+swigp4.cmi:
+	env SWIG_LIB=$(SWIG_LIB_DIR) $(SWIGEXE) -ocaml -co swigp4.ml
+	$(OCC) -I $(OCAMLP4WHERE) -pp "camlp4o pa_extend.cmo q_MLast.cmo" -c swigp4.ml
+extra_obs: swig.cmi swig.cmo swigp4.cmi
```

Comment:
Can you use full English, so extra_objects (I think you mean obs==objects, and a .cmi file is an object).